owslib/iso.py

```diff
@@ -550,8 +553,19 @@ def __init__(self, md=None):
             self.operations = []
             self.operateson = []
         else:
+            self.title=None
             val=md.find(util.nspath_eval('gmd:citation/gmd:CI_Citation/gmd:title/gco:CharacterString', namespaces))
             self.title=util.testXMLValue(val)
+            
+            self.abstract=None
```

Comment:
We don't need to init this here, since `self.abstract` below gets set below no matter what via `util.testXMLValue`